tekton/resources/nightly-release/tasks.yaml

```diff
@@ -57,6 +57,116 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: golang-test
+spec:
+  params:
+  - description: package (and its children) under test
+    name: package
+    type: string
+  - default: ./...
+    description: 'packages to test (default: ./...)'
+    name: packages
+    type: string
+  - default: latest
+    description: golang version to use for tests
+    name: version
+    type: string
+  - default: -race -cover -v
+    description: flags to use for the test command
+    name: flags
+    type: string
+  - default: linux
+    description: running program's operating system target
+    name: GOOS
+    type: string
+  - default: amd64
+    description: running program's architecture target
+    name: GOARCH
+    type: string
+  - default: auto
+    description: value of module support
+    name: GO111MODULE
+    type: string
+  resources:
+    inputs:
+    - name: source
+      targetPath: src/$(inputs.params.package)
+      type: git
+  steps:
+  - env:
+    - name: GOPATH
+      value: /workspace
+    - name: GOOS
+      value: $(inputs.params.GOOS)
+    - name: GOARCH
+      value: $(inputs.params.GOARCH)
+    - name: GO111MODULE
+      value: $(inputs.params.GO111MODULE)
+    image: golang:$(inputs.params.version)
+    name: unit-test
+    resources: {}
+    script: |
+      go test $(inputs.params.flags) $(inputs.params.packages)
+    workingDir: /workspace/src/$(inputs.params.package)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: golang-build-legacy
```

Comment:
I don't have enough context here to know why this is legacy / why I would want to use this over the non-legacy version - I suspect others will run into this as well. (even looking at this diff I can't tell what's different between these).

Let's add a `TaskSpec.Description` to document the differences / guidance on which to use.

Reply:
As per PR description, they are identical.
It's just a trick a I have to use to remove them without breaking nightly builds in the process.